docs/DeveloperGuide.md

```diff
@@ -1364,12 +1377,11 @@ testers are expected to do more *exploratory* testing.
    4. **Other incorrect add commands to try**: `add n/Matthew p/98523146... e/Matt@example.com a/Seletar Lane 12`, `add n/Matthew... p/98523146 e/Matt@example.com a/Seletar Lane 12`, `add n/Matthew p/98523146 e/Matt...@example.com a/Seletar Lane 12`, `add n/Matthew p/98523146 e/Matt@example.com a/Seletar Lane 12...` (where ... represents string that exceeds the character limit). <br> 
       **Expected**: Similar to previous (in Point 3).
 
-<div markdown="span" class="alert alert-primary">
-:information: 
-Character limit:
-1) Name - 0 to 50 characters
-2) Phone - 3 to 20 digits
-3) Email - max 100 characters (at least 2 for domain portion, before @ symbol)
+<div markdown="span" class="alert alert-primary">:information_source:
+**Character limits:** <br>
+1) Name - 0 to 50 characters <br>
+2) Phone - 3 to 20 digits <br>
+3) Email - max 100 characters (at least 2 for domain portion, before @ symbol) <br>
 4) Address - max 100 characters
```

Comment:
I think you missed out character limits for Remarks and Appointment Venue.

Reply:
Character limits added for remark and tag.